plugins/services/src/js/service-configuration/ServiceStorageConfigSection.js

```diff
@@ -31,160 +26,120 @@ class ServiceStorageConfigSection extends ServiceConfigBaseSectionDisplay {
     return "CreateService:ServiceConfigDisplay:App:Storage";
   }
 
+  getVolumeSizeValue(value, type = null) {
+    if (value == null) {
+      return getDisplayValue(value);
+    }
+
+    if (type === "EXTERNAL") {
```

Comment:
Is there a struct for this maybe?

Reply:
I don't understand the question, can you clarify please?

Reply:
We discussed offline, he meant `constants`